compiler/main.bal

```diff
@@ -25,18 +26,18 @@ public function main(string[] filenames, *Options opts) returns error? {
             check showTypes(filename);
             continue;
         }
-        check compileFile(filename, check chooseOutputFilename(filename, opts.outDir));
+        check compileFile(filename, check chooseOutputFilename(filename, opts.outDir), {gcName: opts.gc});
```

Comment:
Please space like this: `{ gcName: opts.gc }`

Reply:
Fixed